packages/app-mobile/components/screens/ConfigScreen/SettingComponent.tsx

```diff
@@ -114,6 +114,33 @@ const SettingComponent: React.FunctionComponent<Props> = props => {
 				</View>
 			</View>
 		);
+	} else if (md.type === Setting.TYPE_BIGINT) {
```

Comment:
Rather than adding a new type (`Setting.TYPE_BIGINT`), it could make sense to define a new subtype. See, for example, [how the FontSearch text input type is implemented](https://github.com/laurent22/joplin/blob/73076bd4b7873fa82ead0183d4c4074a18d60446/packages/app-desktop/gui/ConfigScreen/controls/SettingComponent.tsx#L304) .